feature/metis/conversation/src/test/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/service/TestMetisModificationService.kt

```diff
@@ -0,0 +1,4 @@
+package de.tum.informatics.www1.artemis.native_app.feature.metis.conversation.service
+
+class TestMetisModificationService {
```

Comment:
What is this class for?

Reply:
Must have been left over. I just safe deleted it.